plugins/auth-backend-module-aws-alb-provider/src/helpers.ts

```diff
@@ -79,10 +90,9 @@ export const provisionKeyCache = (region: string, keyCache: NodeCache) => {
     if (optionalCacheKey) {
       return crypto.createPublicKey(optionalCacheKey);
     }
+
     const keyText: string = await fetch(
-      `https://public-keys.auth.elb.${encodeURIComponent(
-        region,
-      )}.amazonaws.com/${encodeURIComponent(header.kid)}`,
+      getPublicKeyEndpoint(region) + header.kid,
```

Comment:
Could you keep the URI encoding for `kid` here, just in case? And in general we avoid string concatenation with `+`, opting for template strings instead since it's a bit more clear when reading.